unit-tests/README.md

```diff
@@ -0,0 +1,268 @@
+# DTS scripts unit tests
```

Comment:
I know it was named this way iniitally, but these are not unit tests really.

How about we at lest rename this to simply `tests` ?